src/cpp/desktop/DesktopMenuCallback.cpp

```diff
@@ -385,13 +395,22 @@ MenuActionBinder::MenuActionBinder(QMenu* pMenu, QAction* pAction) : QObject(pAc
    connect(pMenu, SIGNAL(aboutToHide()), this, SLOT(onHideMenu()));
    pAction_ = pAction;
    keySequence_ = pAction->shortcut();
+   customSequence_ = QKeySequence();
```

Comment:
Nit: since this object has a default constructor, you shouldn't need to explicitly initialize it. 
```suggestion
```